src/app/components/ATIAnalytics/canonical/sendBeaconOperaMini.js

```diff
@@ -0,0 +1,35 @@
+import isOperaProxy from '#app/lib/utilities/isOperaProxy';
+
+export function sendBeaconOperaMini(atiPageViewUrlString) {
+  if (isOperaProxy()) {
+    const xhr = new XMLHttpRequest();
+    xhr.open('GET', atiPageViewUrlString, true);
+    xhr.withCredentials = true;
+    xhr.send();
+  }
+}
+
+export function sendBeaconOperaMiniAsString(atiPageViewUrlString) {
+  let sendBeaconOperaMiniString = sendBeaconOperaMini.toString();
+
+  sendBeaconOperaMiniString = sendBeaconOperaMiniString.replace(
+    'atiPageViewUrlString',
+    '',
+  );
+  sendBeaconOperaMiniString = sendBeaconOperaMiniString.replace(
+    'atiPageViewUrlString',
+    `'${atiPageViewUrlString}'`,
+  );
+
+  const ifExpression = sendBeaconOperaMiniString
+    .split('if (')[1]
+    .split(') {')[0];
+  sendBeaconOperaMiniString = sendBeaconOperaMiniString.replace(
+    ifExpression,
+    `${isOperaProxy.toString()}()`,
+  );
```

Comment:
```suggestion
  sendBeaconOperaMiniString = sendBeaconOperaMiniString.replace(
    'isOperaProxy()',
    `${isOperaProxy.toString()}()`,
  );
```

I assume there's a reason but why can't we just do this?

Reply:
This code is gone now, but we can't directly use the string 'isOperaProxy()', as it's being imported, it gets converted to module import syntax